4 - AnimalShelter/Code/Animal.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace AnimalShelterCode
+{
+    public class Animal
+    {
+
+        public Animal(AnimalType type)
+        {
+            
+            Type = type;
+
+        }
+
+        public AnimalType Type { get; private set; }
+        public int UniqueAnimalId { get; internal set;  }
```

Comment:
Having your properties below your CTOR is not a "bad thing" but I will suggest placing them above your CTOR, they should be the first thing you see when you enter a class. This is because this holds the public state of the class and is often the most needed thing to see / debug / maintain within a class and it makes it easier to enter a class and instantly being able to see "This is the possible state".